interactive/api/controller.py

```diff
@@ -16,15 +16,34 @@
     get_zip_file_path
 from partmc_model.partmc_utils import compress_partmc, get_partmc_zip_file_path
 from shared.rabbit_mq import publish_message
+from acom_music_box import Examples
+from api.request_models import Example
 
 logger = logging.getLogger(__name__)
 
 
 def load_example(example):
```

Comment:
Is this function still needed if the examples are all on the client side now?

Reply:
The examples are still loaded from the server. They are read from the examples packaged in musicbox though